srcpkgs/gns3-gui/template

```diff
@@ -1,12 +1,12 @@
 # Template file for 'gns3-gui'
 pkgname=gns3-gui
 version=2.2.10
-revision=2
+revision=3
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-setuptools python3-psutil python3-jsonschema python3-sentry
  python3-PyQt5-svg python3-PyQt5-websockets python3-sip xterm inetutils-telnet
- gns3-net-converter python3-distro"
+ gns3-net-converter python3-distro python3-attrs python3-pyrsistent"
```

Comment:
Dropping this package is being considered (see #26971 ) so perhaps we can avoid adding pyrsistent as a new package.